feat(eslint-plugin-fiori-tools): lint flex change files (#4706)
SAP/open-ux-tools: packages/eslint-plugin-fiori-tools/docs/rules/sap-enable-paste.md

## Rule Details

### For OData V2 Applications

> For OData V2 applications the `showPasteButton` is a flex change property, so this rule checks if there is a `.change` file for the `showPasteButton` property, where the `newValue` is set to `false`. This property sets whether the paste functionality is enabled in a table.

Reviewer: ```suggestion
For OData V2 applications, the `showPasteButton` is a flex change property, so this rule checks if there is a `.change` file for the `showPasteButton` property, where the `newValue` is set to `false`. This property sets whether paste functionality is enabled in a table.
```

### Warning Message

> #### Incorrect Manifest File in OData V4

Reviewer: ```suggestion
#### Incorrect `manifest.json` File in OData V4
```

The `enablePaste` property is set to `false` which hides the "Paste" button.

> #### Incorrect .change File in OData V2

Reviewer: ```suggestion
#### Incorrect `.change` File in OData V2
```

```json
{
  "content": {
    "property": "showPasteButton",
    "newValue": false
  }
}
```

> The `showPasteButton` property new value is set to `false` which hides the "Paste" button.

Reviewer: ```suggestion
The `newValue` of the `showPasteButton` property is set to `false` which hides the "Paste" button.
```

> #### Correct Manifest File in OData V4

Reviewer: ```suggestion
#### Correct `manifest.json` File in OData V4
```

> The `enablePaste` property is correctly set to `true` or omitted from  `tableSettings`.

Reviewer: ```suggestion
The `enablePaste` property is set to `true` or omitted from  `tableSettings`.
```

> #### Correct .change File in OData V2

Reviewer: ```suggestion
#### Correct `.change` File in OData V2
```

```json
{
  "content": {
    "property": "showPasteButton",
    "newValue": true
  }
}
```

> The `showPasteButton` property new value is set to `false` which hides the "Paste" button.

Reviewer: ```suggestion
The `newValue` of the `showPasteButton` property is set to `false` which hides the "Paste" button.
```

## How to Fix in ODataV4

> To fix the warning, remove it from the `manifest.json` file. The "Paste" button is shown by default.

Reviewer: ```suggestion
To fix the warning, remove the `enableExport` property from the `manifest.json` file. The "Paste" button is shown by default.
```

## How to Fix in OData V2

> To fix the warning, there are two options: either delete the corresponding `.change` file, restoring the default setting, or set `newValue` in the property change file to `true`.

Reviewer: ```suggestion
To fix the warning, there are two options: either delete the corresponding `.change` file, which restores the default setting, or set `newValue` in the property change file to `true`.
```
